smac/runhistory/runhistory2epm.py

```diff
@@ -249,53 +298,63 @@ def _build_matrix(self, run_list, runhistory, instances=None):
             else:
                 X[row, :] = conf.get_array()
             # run_array[row, -1] = instances[row]
-            y[row, 0] = run.cost
-
-        #ensure that minimal value is larger than 0
-        if np.sum(y<=0) > 0:
-                self.logger.warning("Got cost of smaller/equal to 0. Replace by %f since we use log cost." %(constants.MINIMAL_COST_FOR_LOG))
-                y[y<constants.MINIMAL_COST_FOR_LOG] = constants.MINIMAL_COST_FOR_LOG
-        y = np.log10(y)
+            if self.scenario.run_obj == "runtime":
+                if run.status != StatusType.SUCCESS:
```

Comment:
I think this piece of code could be simplified by using the cost instead of `run.time * par_factor`. The cost should already be multiplied with the `par_factor` by the TAE (see [here](https://github.com/automl/SMAC3/blob/master/smac/tae/execute_ta_run.py#L125)). Or did I miss something?